libs/external-router/README.md

```diff
@@ -1,16 +1,9 @@
 # @daffodil/external-router
 
-`@daffodil/external-router` extends `@angular/router` and allows you to render routes defined in external systems like Wordpress, Magento, Contentful, etc, as if you had defined the routes statically in your Angular `Routes`. This is useful when generating user-friendly routes in external applications and resolving them by their user-friendly URI, such as `/sweatshirts` instead of `/category/6` or `category/sweatshirts.`
+`@daffodil/external-router` extends `@angular/router` and allows you to render routes defined in external systems like Wordpress, Magento, Contentful, etc, as if you had defined the routes statically in your Angular `Routes`.
 
-<!-- omit in toc -->
-
-## Table of Contents
-
-- [Installation](#installation)
-- [Usage](#usage)
-- [Drivers](#drivers)
-- [Configuration](/libs/external-router/guides/configuration.md)
-- [Testing](/libs/external-router/guides/testing.md)
+## Overview
+It's useful when generating user-friendly routes in external applications and resolving them by their user-friendly URI, such as `/sweatshirts` instead of `/category/6` or `category/sweatshirts.`
```

Comment:
Moving this sentence like this makes this feel a little disjoint. Can we keep the two sentences together?

Reply:
I think it makes the main description too long.